lldb/test/API/functionalities/postmortem/elf-core/TestLinuxCore.py

```diff
@@ -760,6 +772,61 @@ def test_riscv64_regs(self):
 
         self.expect("register read --all")
 
+    @skipIfLLVMTargetMissing("RISCV")
```

Comment:
If everything is optional in risc-v, it may be better to label these tests by what they contain not what they don't.

riscv64_gpr_only
riscv64_gpr_fpr

Reply:
AArch64 has been saved from this problem by a lot of the new register state being weird enough it doesn't fit in this test very well. So for example scalable matrix extension ended up in `lldb/test/API/linux/aarch64/sme_core_file/TestAArch64LinuxSMECoreFile.py`.

But adding basics like FP here in this file is fine.

Reply:
Just to be on the same page:
Should I move `test_riscv64_regs_gpr`/`test_riscv64_regs_gpr_fpr` tests containing register value checks to separate place (like `lldb/test/API/linux/aarch64/sme_core_file/TestAArch64LinuxSMECoreFile.py` or better to `lldb/test/API/linux/riscv64/fpr_core_file/TestRV64FPRCoreFile.py`). And will keep main tests for RISC-V cores (`test_riscv64_gpr` and `test_riscv64_gpr_fpr`) in `TestLinuxCore.py`, as they are using `do_test()` infrastructure from this file? Or you mean something else?

Reply:
I'm saying that if risc-v grows a lot of strange register state, it will be better served by separate test cases.

For this PR, just consider renaming the test functions here so that they list what they are testing. So riscv64_gpr, as opposed to to riscv64_nofpr.

Reply:
As an example of what I mean about future extensions being "too weird" -

In the new Arm extension support I'm doing right now, I wrote a program file that can be used for live processes, and to generate a core file. This keeps all tests for that extension in one place. This helps because I have to use weird assembly to trigger these things, and I only have to write that once. And I don't have to put that in the standard corefile tests with some #ifdefs which get confusing.

With floating point, you can write normal C code to set values, which can be included easily in a program file that's quite generic.

Reply:
> For this PR, just consider renaming the test functions here so that they list what they are testing. So riscv64_gpr, as opposed to to riscv64_nofpr.

Or if you really want to be super clear: `riscv_gpr_only`